### ISSUE TITLE
Unable to Install Terminal (with Fix)

### ISSUE THREAD

**Issue by xeu100** [elementary/appcenter#1470]
## What Happened
* Reinstalling/Updating Terminal in App Center
* causes the error `pantheon-terminal: Depends: io.elementary.terminal but it is not going to be installed`

## Expected Behavior
Installs or updates

## Steps to Reproduce
1. Press the Free button to install or press the update button

### Logs
N/A

## Platform Information
elementaryOS 6

**Comment by mrgian** [comment]
I don't know if this is an useful workaround for someone but I got Terminal installed by doing this:
open Files, go to /usr/bin, right click gnome-terminal, run and type sudo apt install io.elementary.terminal

**Comment by xeu100** [comment]
It's a good workaround, we're probably just waiting for the pantheon-terminal package to get fixed for `focal` on their ppa.

**Comment by danrabbit** [comment]
@xeu100 Can you be a bit more specific about steps to reproduce? Which build did you install from? Was Terminal not installed out of the box or did you uninstall it?

**Comment by xeu100** [comment]
> @xeu100 Can you be a bit more specific about steps to reproduce? Which build did you install from? Was Terminal not installed out of the box or did you uninstall it?

Sure! I had this issue on elementary-live with a build from source (yesterday, I've been told someone reproduced it with source today as well). I had also experienced this issue using meisenzahl's upgrade tool from 5.1.7. The live USB did not have it at all while the upgrade did. The package `io.elementary.terminal` works fine, but AppCenter wants to use `pantheon-terminal` which conflicts with it. I've tried a few things (and a few different tools) to see how to resolve the conflict but it seems that `pantheon-terminal` both conflicts and depends on `io.elementary.terminal`
